### PR TITLE
addsWishListStyling

### PR DESCRIPTION
**Pull request by ghost**
for your review. thanks!

### CODE REVIEW

File: src/components/WishlistPanel/WishlistPanel.css [hunk 1]
```diff
@@ -37,3 +37,97 @@
...
+  background: white;
+  padding: 10px 40px;
+  margin: 50px auto;
+  max-width: 800px;
+  width: 70%;
+  border-radius: 10px;
+  -webkit-box-shadow: -1px 1px 39px -7px rgba(0,0,0,0.75);
+  -moz-box-shadow: -1px 1px 39px -7px rgba(0,0,0,0.75);
+  box-shadow: -1px 1px 39px -7px rgba(0,0,0,0.75);
+}
```
Comment (astr0-4) [inline src/components/WishlistPanel/WishlistPanel.css]: we don't need the shadow or the border radius for the wishlist

File: src/components/WishlistPanel/WishlistPanel.css [hunk 2]
```diff
@@ -37,3 +37,97 @@
   max-width: 200px;
   min-width: 120px;
 }
+
+.product-wishList {
+  background: white;
```
Comment (astr0-4) [inline src/components/WishlistPanel/WishlistPanel.css]: on the image from the trello board, the background is the same colour as the wishlist panel, so we can remove this

File: src/components/WishlistPanel/WishlistPanel.css [hunk 3]
```diff
@@ -37,3 +37,97 @@
...
+
+  .product:first-of-type {
+    border-top: 1px solid #747474;
+  }
+
+  /*.product img {
+    width: 25%;
+    padding: 0;
+  }*/
+
```
Comment (astr0-4) [inline src/components/WishlistPanel/WishlistPanel.css]: if you don't need this code, just delete it

File: src/components/WishlistPanel/WishlistPanel.css [hunk 4]
```diff
@@ -37,3 +37,97 @@
...
+  .product .text-wrapper {
+    margin: 0;
+    /*padding: 0;*/
```
Comment (astr0-4) [inline src/components/WishlistPanel/WishlistPanel.css]: same here

File: src/components/WishlistPanel/WishlistPanel.css [hunk 5]
```diff
@@ -37,3 +37,97 @@
...
+    width: 25%;
+    padding: 0;
+  }
+
+  
+
+@media (max-width: 1024px) {
+  .wishlist-panel {
+    
+  }
```
Comment (astr0-4) [inline src/components/WishlistPanel/WishlistPanel.css]: can we remove this `.wishlist-panel` section here?

**Review by astr0-4 — CHANGES_REQUESTED** [review]
this looks awesome @holdersimone! I love how responsive it is!!!!! 💯  💯  
Just some small changes: 
in the image in the trello board, the styling is a bit different. We don't need the white background, or the corner radius and shadow for the wishlist products. There is also a little "x" button in the top right corner, but I think it's fine to leave the whole product as selectable, maybe just add the x in the corner? 

You also might need to reduce the padding on the left to make more room, and maybe make the image a bit smaller? The text is still a little cramped.

**Comment by astr0-4** [comment]
The designs from Trello:
screen shot 2017-07-30 at 9 08 59 am

**Comment by astr0-4** [comment]
@holdersimone this is looking awesome!!! Only 1 small thing is that when the alcohol name is more than one line, the "X" button is pushed up: 
screen shot 2017-07-30 at 6 50 28 pm

**Comment by jocelynjeffrey** [comment]
Looks great!

**Review by jocelynjeffrey — COMMENTED** [review]
Hey @holdersimone  - it's looking good! One thing I noticed is that we're positioning a lot of elements in the wishlist panel with 'left: 30px'. Rather than repeating that bit of code on all of the elements, we can put padding directly on the wishlist panel itself. That way anything we drop into the panel later will automatically line up with everything we've already got in there. I've put in a screen shot of what I mean - let me know what you think!

screen shot 2017-07-30 at 7 07 01 pm

If you decide to put padding on the entire panel, you can remove the `left: 30px` on the close button, `margin-left` on the h2, and set padding to 0px on the ul.

I think you can fix the bug @astr0-4 found by moving the close button code out of the wl-text-wrapper and up to the li level. Once it's there, you can position it relative to the entire li instead of the text-wrapper.

**Comment by ghost** [comment]
Changes have been made. Thanks! 😊

**Review by astr0-4 — APPROVED** [review]

**Comment by astr0-4** [comment]
wooo looks amazing @holdersimone!!! 💯  🎉  👌

**Comment by jocelynjeffrey** [comment]
Hi @holdersimone - the styles are looking really good on mobile! I think they still need to be applied to screens larger than 1024 - I lose the padding on my desktop: 

screen shot 2017-07-31 at 1 01 51 pm

**Review by jocelynjeffrey — APPROVED** [review]

**Pull request merged**